machines/flush-cache.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+
+
+  friendlyName: 'Flush cache',
+
+
+  description: 'Flushes the cache, removing all data from it.',
```

Comment:
@luislobo this is really minor, but we could we change that to say:

> Flush the cache, removing all data from it.

(that way it's in the "imperative mood".  Other than for consistency, the reason I personally care so much is that I've done some work generating automated paragraphs from machines/circuits in the past, and we'll likely do more of that in the future.  So having predictable sentence structure gives everyone working with machines lots of options for some pretty awesome tooling)